repository/packages/azure/psql/0.16.0.yml

```diff
@@ -0,0 +1,135 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  creationTimestamp: null
+  name: psql.azure.references.services.apps.tanzu.vmware.com.0.16.0
+spec:
+  refName: psql.azure.references.services.apps.tanzu.vmware.com
+  releasedAt: "2022-09-13T12:09:43Z"
+  template:
+    spec:
+      deploy:
+      - kapp: {}
+      fetch:
+      - imgpkgBundle:
+          image: ghcr.io/vmware-tanzu-labs/trp-azure-psql@sha256:8bdfdf7a487a9a7dc7ccc60e3213d518ee470111fa9e1a3561eee870fb4957d8
```

Comment:
nit: To not loose track of the different images flying around, I'd prefer to have them all in ghcr.io/vmware-tanzu/tanzu-application-platform-reference-service-packages and below.

See also https://github.com/orgs/vmware-tanzu/packages?tab=packages&q=reference

P.S.: We have some packages in `ghcr.io/vmware-tanzu/tanzu-application-platform-reference-packages`, but they were created before the repo has been renamed and we decided to keep the around (at least for a bit) because the stuff was already "released".

Reply:
Totally agree. We used the other repo as a staging ground, so while we are testing we do not pollute the publicly documented repo.

Reply:
We have not yet updated this to reflect the "final" location.
Keeping this comment open to be clear.

@matteo-magni @bzhtux

Reply:
Done